examples/glue_benchmark/glue.py

```diff
@@ -49,6 +49,12 @@
     "Learning rate",
 )
 
+flags.DEFINE_string(
+    "tpu_name",
+    None,
+    "The name of TPU to connect to. If you only have one TPU, use `local`",
```

Comment:
Maybe add a line making it clear this can be turned off?

The name of TPU to connect to. If None, a TPU strategy will not be used. If you only have one TPU, use `local`.